lnrpc/routerrpc/router.proto

```diff
@@ -161,6 +161,22 @@ message SendToRouteRequest {
 
     /// Route that should be used to attempt to complete the payment.
     lnrpc.Route route = 2;
+
+    /**
+    If non-zero, the total amount in satoshis being sent as part of a larger
+    multi-path payment. The caller is responsible for ensuring subpayments to
+    the same node and payment_hash are equal or greater than total_amt_msat. The
+    same total_amt_msat must be used on all subpayments.
+    */
+    int64 total_amt_msat = 3;
+
+    /**
+    If total_amt_msat is non-zero, a unique, random identifier used to
+    authenticate the sender as the intended payer of a multi-path payment. The
+    payment_addr must be the same for all subpayments, and match the
+    payment_addr provided in the receiver's invoice.
+    */
+    bytes payment_addr = 4;
```

Comment:
Does it need to be added here or into the message `Hop`? Previously, `Route` was a fully specified route that only needed to be launched. Now there are external values required too. I am also thinking about returning a route, for example after a successful payment. Then those external values would be required too for a complete view on what was executed.